spec.md

```diff
@@ -1663,26 +1658,26 @@ A Controller plugin MUST implement this RPC call if plugin has `EXPAND_VOLUME` c
+If plugin has `EXPAND_VOLUME` node capability, then `NodeExpandVolume` MUST be called after successful `ControllerExpandVolume` and `fs_resize_required` in `ControllerExpandVolumeResponse` is `true`.
 
-If the plugin has only `VolumeExpansion.OFFLINE` expansion capability, `ControllerExpandVolume` MUST be called after `ControllerUnpublishVolume` is called and is successful if the volume is currently published to a node.
+If the plugin has only `VolumeExpansion.OFFLINE` expansion capability and volume is currently controller-published to a node , `ControllerExpandVolume` MUST be called ONLY after succesful `ControllerUnpublishVolume`.
 
-`ControllerExpandVolume` RPC call MUST be idempotent and if size of underlying volume already meets requested capacity, the plugin MUST respond with successfull response.
+`ControllerExpandVolume` RPC call MUST be idempotent and if size of underlying volume already meets requested capacity, the plugin MUST respond with successful response.
 
 Examples:
 * Offline Volume Expansion:
-  Given a ElasticSearch process that runs on Azure Disk and needs more space.
-  - The administrator takes the elasticsearch server offline by stopping the workload and CO calls `ControllerUnpublishVolume`.
+  Given an ElasticSearch process that runs on Azure Disk and needs more space.
+  - The administrator takes the Elasticsearch server offline by stopping the workload and CO calls `ControllerUnpublishVolume`.
   - The administrator requests more space for the volume from CO.
   - The CO in turn first makes `ControllerExpandVolume` RPC call which results in requesting more space from Azure cloudprovider for volume-id that was being used by ElasticSearch.
   - Once `ControllerExpandVolume` is completed and successful, the CO will inform administrator about it and administrator will resume the ElasticSearch workload.
-  - On node where ElasticSearch workload is scheduled when workload starts after `NodeStageVolume` the CO calls `NodeExpandVolume`.
+  - The node on which ElasticSearch workload is scheduled the CO calls `NodeExpandVolume` after calling `NodeStageVolume`.
   - Calling `NodeExpandVolume` on volume results in expanding the underlying file system and added space becomes available to workload when it starts up.
 * Online Volume Expansion:
   Given a Mysql server running on Openstack Cinder and needs more space.
   - The administrator requests more space for volume from the CO.
   - The CO in turn first makes `ControllerExpandVolume` RPC call which results in requesting more space from Openstack Cinder for given volume.
-  - The node where mysql workload is running the CO makes `NodeExpandVolume` RPC call while volume is in-use using path where volume is staged.
+  - The node on which mysql workload is running the CO makes `NodeExpandVolume` RPC call while volume is in-use using path where volume is staged.
```

Comment:
suggest: Upon the node where the Mysql workload is running...

Reply:
these were addressed by @jdef PR. marking as resolved.